### DISCUSSION TITLE
Toha theme not in Hugo website.

### DISCUSSION THREAD

**Discussion by BernatBC** [hugo-themes/toha#865] [General]
Hi @hossainemruz , 
I was looking into other hugo themes to get some ideas, and in https://themes.gohugo.io/ I couldn't find the Toha theme. I don't know how long has it been since it was removed. I just wanted to ask if you are aware that is not in the hugo website.

**Comment by hossainemruz** [comment]
The theme is still there. However, it seems their build system is broken since last month.
https://github.com/gohugoio/hugoThemesSiteBuilder/actions

**Comment by BernatBC** [comment]
I've seen that the build system is working correctly, but still cannot find the theme on the hugo site

**Reply by hossainemruz** [reply]
I am not really sure why the theme is not on the theme site. The theme site is using old repository URL. I have sent a PR to fix that. Not merged yet.
https://github.com/gohugoio/hugoThemesSiteBuilder/pull/400

**Comment by hossainemruz** [comment]
Created a new PR with updated URL. The build is now passing. I hope they will merge the PR.
https://github.com/gohugoio/hugoThemesSiteBuilder/pull/419

**Comment by hossainemruz** [comment]
Finally, the theme is on Hugo website.
https://themes.gohugo.io/themes/toha/
